src/actions/productsActions.js

```diff
@@ -96,7 +96,7 @@ export const addProduct = newProd => dispatch => {
       });
       dispatch(getVendorBatch(res.data.vendorId));
       dispatch(getCategoryBatch(res.data.categoryId));
-      dispatch(setProductLoaded());
+      //dispatch(setProductLoaded());
```

Comment:
Can you remove commented out code -- or comment to say why it's still there>?

Reply:
I'm going to fix that up now. I commented that out to test the state being updated. It was breaking the page.